edspc/magento2
# Remove recursion in location of static files.

### Description (*)
The default NGINX config contains a possible bug in location `static`, that leads to infinite recursion.
This PR remove it.

### Manual testing scenarios (*)
1. Enable [Static content signing](https://devdocs.magento.com/guides/v2.4/config-guide/cache/static-content-signing.html#static-content-signing).
2. Try to open URL like [MAGE_URL]/static/version-example

**Expected**: 404 page
**Actual**: 500 error

### Contribution checklist (*)
